resources/kcp/values.yaml

```diff
@@ -10,8 +10,8 @@ global:
       dir: control-plane/
       version: "v20240613-0d0eb85f"
     subscription_cleanup_job:
-      dir:
-      version: "v20240129-70857389"
+      dir: "prod/"
```

Comment:
Won't this result in `europe-docker.pkg.dev/kyma-project/prod/prod/subscription-cleanup-job`?

The actual path is `europe-docker.pkg.dev/kyma-project/prod/subscription-cleanup-job`

Reply:
Right, sorry overlooked that `/prod` is already part of the registry name